src/storage/v3/indices.hpp

```diff
@@ -59,7 +60,7 @@ class LabelIndex {
   void UpdateOnAddLabel(LabelId label, Vertex *vertex, const Transaction &tx);
 
   /// @throw std::bad_alloc
-  bool CreateIndex(LabelId label, VerticesSkipList::Accessor vertices);
+  bool CreateIndex(LabelId label, VertexContainer &vertices);
```

Comment:
The issue with `LabelIndex` is that there is no order, and using any other structures will slow it down since lookup is important on update/delete. I would just keep `SkipList` for now